software-engineer-I/fc-1/javascript/modulo-3/ex-7.js

```diff
@@ -0,0 +1,15 @@
+const blueSky = false;
+const niceWeather = false;
+const windy = false;
+
+const checkNiceDay = (blueSky, niceWeather, windy) => {
+    if ((blueSky || niceWeather) && !windy) {
+        return "Ótimo dia para um pequenique";
+    } else if ((!blueSky || !niceWeather) && windy) {
+        return "Péssimo dia para um pequenique";
+    } else {
+        return "Não é possível definir";
```

Comment:
o else já é `"Péssimo dia para um pequenique";`

Pois se o dia (não está azul ou não está numa temperatura  boa) e está ventando, não da para sair

Reply:
mas e se for tudo false?

Reply:
((isBlueSky && isNiceWeather) && !isWindy)